app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,101 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.Task;
+import com.codesoom.assignment.TaskList;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.io.IOException;
+import java.util.NoSuchElementException;
+
+/**
+ * Handle requests about task.
+ */
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    TaskList taskList = new TaskList();
+    JsonParser jsonParser = new JsonParser();
+
+    @GetMapping
+    public String list() {
+        try {
+            return jsonParser.toJson(taskList.getTaskList());
+        } catch (IOException e) {
+            throw new ResponseStatusException(
+                    HttpStatus.NOT_FOUND, "Parsing error"
+            );
+        } catch (Exception e) {
+            throw new ResponseStatusException(
+                    HttpStatus.INTERNAL_SERVER_ERROR, "Unknown error"
+            );
+        }
+    }
```

Comment:
jsonParser를 사용하는 이유는 무엇인가요? 이렇게만 해도 될거에요

```suggestion
    @GetMapping
    public List<Task> list() {
       return taskList.getTaskList()
    }
```